openedx/features/edly/templates/edly/edx_ace/handoutchangesnotification/email/body.html

```diff
@@ -0,0 +1,53 @@
+{% extends 'ace_common/edx_ace/common/base_body.html' %}
+
+{% load i18n %}
+{% load static %}
+{% block content %}
+<table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
+    <tr>
+        <td>
+            <h1>
+                {% blocktrans %}
+                    Handout of course {{ course_name }} on {{ site_name }} has been updated
```

Comment:
`Handout of course {{ course_name }} on {{ site_name }} has been updated` -> 
`Handouts of course {{ course_name }} on {{ site_name }} have been updated`

Reply:
Done